exe-unit/components/http-proxy/src/gsb_to_http.rs

```diff
@@ -0,0 +1,106 @@
+use crate::headers::Headers;
+use crate::message::GsbHttpCallMessage;
+use crate::response::GsbHttpCallResponseEvent;
+use async_stream::stream;
+use chrono::Utc;
+use futures_core::stream::Stream;
+use tokio::sync::mpsc;
+
+#[derive(Clone, Debug)]
+pub struct GsbToHttpProxy {
+    pub base_url: String,
+}
+
+impl GsbToHttpProxy {
+    pub fn pass(
+        &mut self,
+        message: GsbHttpCallMessage,
+    ) -> impl Stream<Item = GsbHttpCallResponseEvent> {
+        let url = format!("{}{}", self.base_url, message.path);
+
+        let (tx, mut rx) = mpsc::channel(24);
```

Comment:
Not sure why you allocate 24 elements?
This channel will be used only once as I understand code. Am I right?